ui/helpers/urlHelper.js

```diff
@@ -51,6 +51,14 @@ export const getLogViewerUrl = function getLogViewerUrl(job_id, repoName, line_n
   return line_number ? `${rv}&lineNumber=${line_number}` : rv;
 };
 
+export const getRawLogUrl = function getRawLogUrl(url, value) {
```

Comment:
This URL seems to be for the WPT viewer rather than the raw log, so I think a different function name might be best?